plugins/AccountStats.py

```diff
@@ -118,36 +118,48 @@ def notify_stats(self):
             return
 
         self.earnings = {}
+        output = ''
 
         # Today's earnings
         cursor = self.db.execute(DB_GET_TODAYS_EARNINGS)
-        output = ''
-        if cursor.rowcount > 0:
-            for row in cursor:
+        row = cursor.fetchone()
+        if row is not None:
+            while row is not None:
                 output += self.format_value(row[0]) + ' ' + str(row[1]) + ' Today\n'
                 if row[1] not in self.earnings:
                     self.earnings[row[1]] = {}
                 self.earnings[row[1]]['todayEarnings'] = row[0]
+                row = cursor.fetchone()
         else:
             output += 'None Today\n'
         cursor.close()
 
```

Comment:
# Today's earnings (Using fetchall to handle multiple rows)
rows = cursor.fetchall()                
if len(rows) > 0:
    for row in rows:
        output += self.format_value(row[0]) + ' ' + str(row[1]) + ' Today\n'
        if row[1] not in self.earnings:
        self.earnings[row[1]] = {}
        self.earnings[row[1]]['todayEarnings'] = row[0]				
    else:
        output += 'None Today\n'
cursor.close()